corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/entries_spec.js

```diff
@@ -228,6 +228,24 @@ describe('Entries', function () {
         assert.sameMembers(entry.answer(), [2]);
     });
 
+    it('Should retain MultiSelectEntry when choices change', function () {
+        questionJSON.datatype = Const.MULTI_SELECT;
+        questionJSON.choices = ['a', 'b'];
+        questionJSON.answer = [1, 2]; // answer is based on a 1 indexed index of the choices
```

Comment:
A comment like this with where the new `rawAnswer` computeds are defined would be helpful.